src/resources/Ingresses/IngressStatus.js

```diff
@@ -0,0 +1,107 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ConditionList } from 'shared/components/ConditionList/ConditionList';
+import { EMPTY_TEXT_PLACEHOLDER } from 'shared/constants';
+import { spacing } from '@ui5/webcomponents-react-base';
+import './IngressStatus.scss';
+
+export const IngressStatus = ({ resource }) => {
+  const { t } = useTranslation();
+  const ingresses = useMemo(() => {
+    return resource?.status?.loadBalancer?.ingress?.map(ingress => ingress);
+  }, [resource]);
+
+  return ingresses ? (
+    <ConditionList
+      className="load-balancers"
+      conditions={ingresses.map(ingress => ({
+        header: {
+          titleText: ingress.hostname ? (
+            <div>
+              <span
+                style={{ ...spacing.sapUiTinyMarginEnd }}
+                className="title bsl-has-color-status-4"
+              >
+                {`${t('ingresses.labels.host-name')}:`}
+              </span>
+              {`${ingress.hostname}`}
+            </div>
+          ) : (
+            <div>
+              <span
+                style={{ ...spacing.sapUiTinyMarginEnd }}
+                className="title bsl-has-color-status-4"
+              >
+                {`${t('ingresses.labels.ip')}:`}
+              </span>
+              {`${ingress.ip}`}
+            </div>
+          ),
+        },
+        customContent: [
+          ...(ingress.hostname
```

Comment:
as far as I see we can remove those spread operators here and below and simply not place the resulting object inside of an array